db/migrate/20240117180824_latin1_to_utf8.rb

```diff
@@ -0,0 +1,15 @@
+class Latin1ToUtf8 < ActiveRecord::Migration[7.1]
+  def up
+    Article.connection.
+      execute("ALTER TABLE articles CONVERT TO CHARACTER SET utf8mb4;")
+    Sequence.connection.
+      execute("ALTER TABLE sequences CONVERT TO CHARACTER SET utf8mb4;")
```

Comment:
1. Should I have separate migrations for Article and Sequence?
2. Is is possible to do this without SQL, for instance using `:options`? I couldn't figure out how.